templates/user/notification/notification_subscriptions.tmpl

```diff
@@ -1,15 +1,15 @@
 {{template "base/head" .}}
 <div role="main" aria-label="{{.Title}}" class="page-content user notification">
 	<div class="ui container">
-		<div class="ui top attached tabular menu">
+		<div class="ui compact small menu small-menu-items">
 			<a href="{{AppSubUrl}}/notifications/subscriptions" class="{{if eq .Status 1}}active {{end}}item">
 				{{ctx.Locale.Tr "notification.subscriptions"}}
 			</a>
 			<a href="{{AppSubUrl}}/notifications/watching" class="{{if eq .Status 2}}active {{end}}item">
 				{{ctx.Locale.Tr "notification.watching"}}
 			</a>
 		</div>
-		<div class="ui bottom attached active tab segment">
+		<div class="ui top attached active tab segment">
```

Comment:
https://fomantic-ui.com/modules/tab.html

`tab` should be used with `tabular menu`, but there is no `tabular menu` now?

Reply:
Likely some JS is still bound to the `tab` class, but I can check if I can remove there as well.

Reply:
This case is fixed in https://github.com/go-gitea/gitea/pull/30707/commits/3de13aa93fe676b8ff4a509bf1d0d0fc1b86ceee but there are more.

Reply:
Damn this htmx crap is hard to debug.

Reply:
I'm not going ahead with the `tab` class removal, it's required for the tab-to-content association to work and we can think about  this again once we remove the fomantic `tab` module.